### DISCUSSION TITLE
HTTPS support ?

### DISCUSSION THREAD

**Discussion by trungkien210493** [grafana/grafana-infinity-datasource#73] [Q&A] [answered]
Hi,
I tried to use http request and it works properly. But when I tried with HTTPS (I declared the user/pass in datasource and disable the TLS verification) but I can't not get return data. Does the datasource support HTTPS ?
![image](https://user-images.githubusercontent.com/43126932/117643372-85adff80-b1b2-11eb-9e4d-a2daa00fbe9b.png)

**Comment by yesoreyeram** [comment, marked as answer]
Version 0.7 doesn't support skip tls verify yet. You can use 0.6 version which supports skip tls verfiy.

**Comment by meghanan123** [comment]
Does infinity data source support HTTPS in the URL? What is the update as of today?
